apps/server/src/modules/registration/api/service/registration-feature.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { FeatureDisabledLoggableException } from '@shared/common/loggable-exception';
+import { RegistrationConfig } from '../../registration.config';
+
+@Injectable()
+export class RegistrationFeatureService {
+	constructor(private readonly configService: ConfigService<RegistrationConfig, true>) {}
+
+	public checkFeatureRegistrationEnabled(): void {
+		if (!this.configService.get('FEATURE_REGISTRATION_ENABLED', { infer: true })) {
+			throw new FeatureDisabledLoggableException('FEATURE_REGISTRATION_ENABLED');
```

Comment:
I like the idea to extract the check... but I think it should rather be a little util ... and I think it should not be a loggableException but rather something like http FORBIDDEN...

Reply:
yes I oriented on the room permission service and thought it would be good practice to follow the same